app/workers/insert_lab_records_worker.rb

```diff
@@ -1,22 +1,18 @@
 class InsertLabRecordsWorker
   include Sidekiq::Worker
 
-  def perform(lab_record_import_id, lab_records)
+  def perform(lab_record_import_id)
     Rails.logger.info lab_record_import_id
```

Comment:
Remove this?

Reply:
It's safe to remove that parameter because it's old `lab_records_attributes`, the one that was a JSON and was now replaced with an attachment